packages/melos/lib/src/commands/version.dart

```diff
@@ -309,10 +309,15 @@ mixin _VersionMixin on _RunMixin {
       return;
     }
 
+    final pendingPackageUpdatesCount = updateDependentsVersions
+        ? pendingPackageUpdates.length
+        : pendingPackageUpdates
+            .where((update) => update.reason != PackageUpdateReason.dependency)
+            .length;
```

Comment:
It looks to me like the `pengingPackageUpdates` should be the list that is excluding the the packages, not the count variable

Reply:
Looks like that! I saw using later on `updateDependentsVersions` e.g. in `_gitCommitChanges` and thought it was for a reason but does not look like that. Tested